src/utils/render.tsx

```diff
@@ -11,6 +11,10 @@ import Clipboard from '../components/Clipboard';
 import { AttributeDefinition, AttrbuteDataType, ConseilFunction } from 'conseiljs';
 import { truncateHash, formatNumber } from './general';
 
+interface Attributes extends AttributeDefinition {
+  valueMap?: Record<string, string>
```

Comment:
This change should just be made in ConseilJS

Reply:
This will be in the next [0.4.x series](https://github.com/Cryptonomic/ConseilJS/blob/release/040/src/types/conseil/MetadataTypes.ts#L45) beta release.

Reply:
👍